Tests/FluentAssertions.Equivalency.Specs/FluentAssertions.Equivalency.Specs.csproj

```diff
@@ -2,7 +2,7 @@
 
   <!-- To reduce build times, we only enable analyzers for the newest TFM -->
   <PropertyGroup>
-    <TargetFrameworks>net47;net5.0;netcoreapp2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>net47;net5.0;net6.0;netcoreapp2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
🤔 Can't we remove `net5.0`? Same in the `FluentAssertions.Specs.csproj`.

Reply:
Yes we can, we added only added net5.0 to test records.